composer.json

```diff
@@ -16,12 +16,19 @@
         }
     ],
     "require": {
-        "php": ">=5.5.0",
-        "guzzlehttp/guzzle": "~6.0"
+        "ext-json": "*",
+        "php": ">=7.1",
+        "guzzlehttp/guzzle": "~6.0",
+        "illuminate/support": "^5.5"
     },
     "require-dev": {
         "phpunit/phpunit": "~4.8.35"
     },
+    "config": {
```

Comment:
Should we need to define the `config` block in `composer.json`?

I think it's up to developers to build their current development environment.